tests/test_nn_models.py

```diff
@@ -18,11 +18,7 @@
 
 
 def test_model_load():
-    config = ldp.nn.LMConfig(
-        model="gpt2",
-        load_args={"attn_implementation": "eager"},
-        dtype=ldp.nn.TorchDType.fp32,
-    )
+    config = ldp.nn.LMConfig(model="gpt2")
```

Comment:
Jw, what made you change this?

Reply:
I used to have "eager" to avoid the FlashAttention2 default in tests. Then I modified that default to only apply if a GPU is being used. This test is CPU-only, so it becomes superfluous. I don't remember why I had a dtype specified